### ISSUE TITLE
Allow forced colorization via an option

### ISSUE THREAD

**Issue by Kshitiz-Sharma** [nicoulaj/rainbow#128]
Log files are usually viewed on the console using tools like less. 

Similar to rainbow, grep omits colors from its output if the destination is not console. However, it allows forcing it using `--color=always`. This combined with the `-R` switch in less allows viewing the output with colors and have the scroll, search options of less.

At the moment rainbow seems to want to dump the entire file to console. Unless this is mentioned in the documentation somewhere, and I've missed it.

**Comment by joelpurra** [comment]
@Kshitiz-Sharma: I agree, a [`--color[=WHEN]`](https://linux.die.net/man/1/grep) flag (`never`, `always`, default `auto`) would be great.

As a workaround, you can use the [undocumented environment variables `RAINBOW_ENABLE_STDOUT`/`RAINBOW_ENABLE_STDERR`](https://github.com/nicoulaj/rainbow/blob/2.7.1/rainbow/__init__.py#L43-L44). Below using [`hexdump`](https://en.wikipedia.org/wiki/Hex_dump) to verify the byte output instead of screenshots with color. Replace `hexdump -c` with `cat` to get the "real" colorized output.

---

#### With pipe from `rainbow`, piped to `hexdump`.

Note that the `b` would colored red without `hexdump`.

```shell
RAINBOW_ENABLE_STDOUT=true rainbow --red 'b' echo 'abc' | hexdump -c
```

```text
0000000   a 033   [   3   1   m   b 033   [   3   9   m   c  \n 033   [
0000010   0   m                                                        
0000012
```

#### With pipe through `rainbow`, piped to `hexdump`.

```shell
echo 'abc' | RAINBOW_ENABLE_STDOUT=true rainbow --red 'b' | hexdump -c
```
Note that the `b` would colored red without `hexdump`.

```text
0000000   a 033   [   3   1   m   b 033   [   3   9   m   c  \n 033   [
0000010   0   m                                                        
0000012

```

#### Watching a command, with output piped to `rainbow`

My use-case is `watch --color`. Note that [`--color`](https://gitlab.com/procps-ng/procps/-/blob/master/watch.1#L55-56) is not available in all `watch` versions, but at least in [the `apt` package `procps`](https://packages.debian.org/search?keywords=procps) from [`procps-ng`](https://gitlab.com/procps-ng/procps/) v3.3.15.

Colorization works only with `RAINBOW_ENABLE_STDOUT=true`. Note that the output would be garbled without `--color`.

```shell
watch --color "echo 'abc' | RAINBOW_ENABLE_STDOUT=true rainbow --red 'b'"
```
